docs/features/dns-cache.rst

```diff
@@ -31,7 +34,7 @@ of next_node.name may be resolved even without upstream connectivity.
   },
 
   next_node = {
-    name = 'nextnode',
+    name = 'nextnode.yourdomain.net',
```

Comment:
Not every community uses a searchdomain. My intention was to keep it as universal as possible. For me the doc should contain examples which are working for the most of the users. searchdomain would confuse some....